src/main/java/com/ntnghia/task/entity/Task.java

```diff
@@ -0,0 +1,28 @@
+package com.ntnghia.task.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "task")
+public class Task {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+    @Column(name = "title")
```

Comment:
Không cần cái này nếu mình không có nhu cầu đổi tên cột

```suggestion
    @Column
```

Reply:
Done